setup.py

```diff
@@ -48,22 +48,12 @@ def is_platform_mac():
     _CYTHON_INSTALLED = False
     cythonize = lambda x, *args, **kwargs: x  # dummy func
 
-# The import of Extension must be after the import of Cython, otherwise
-# we do not get the appropriately patched class.
-# See https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html # noqa
-from distutils.extension import Extension  # isort:skip
-from distutils.command.build import build  # isort:skip
-
 if _CYTHON_INSTALLED:
-    from Cython.Distutils.old_build_ext import old_build_ext as _build_ext
-
-    cython = True
-    from Cython import Tempita as tempita
+    from Cython import Tempita
+    from Cython.Distutils.build_ext import build_ext as _build_ext
```

Comment:
Hmm we should do this but might want to be careful - any idea what the difference is with the new build_ext command?

Reply:
I'll look into this.

Reply:
Looking at the cython [source code](https://github.com/cython/cython/blob/c3121126b041d4e7e2aaa590693689ae828f3ce8/Cython/Distutils/build_ext.py#L5). The new build_ext is just 
`setuptools.command.build_ext` or `distutils.command.build_ext`. But I'm not sure how it differs from the old_build_ext.